scripts/prepare.mjs

```diff
@@ -7,4 +7,4 @@ if (process.env.CI || process.env.DOCKER_BUILD) {
 	process.exit(0);
 }
 
-execSync('./node_modules/.bin/lefthook install', { stdio: 'inherit' });
+execSync('npx lefthook install', { stdio: 'inherit' });
```

Comment:
We should use `pnpm` instead of `npm` to use the already installed version of `lefthook`
```suggestion
execSync('pnpm lefthook install', { stdio: 'inherit' });
```